core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/signing/eth2/Eth2SigningRequestBody.java

```diff
@@ -44,6 +45,7 @@ public Eth2SigningRequestBody(
       @JsonProperty("signingRoot") final Bytes signingRoot,
       @JsonProperty("fork_info") final ForkInfo fork_info,
       @JsonProperty("block") final BeaconBlock block,
+      @JsonProperty("beacon_block") final BlockRequest blockRequest,
```

Comment:
It is a bit confusing having block and beacon_block in the request. Not sure there is much we can do about now that Teku has implemented this though